js/privatebin.js

```diff
@@ -3709,21 +3771,18 @@ jQuery.PrivateBin = (function($, RawDeflate) {
         function removeAttachment(event)
         {
             // if custom attachment is used, remove it first
-            if (!$customAttachment.hasClass('hidden')) {
-                AttachmentViewer.removeAttachment();
-                $customAttachment.addClass('hidden');
-                $fileWrap.removeClass('hidden');
-            }
+            me.clearCustomAttachment();
 
             // in any case, remove saved attachment data
-            AttachmentViewer.removeAttachmentData();
+            AttachmentViewer.removeAttachment();
 
             clearAttachmentInput();
-            AttachmentViewer.clearDragAndDrop();
 
             // pevent '#' from appearing in the URL
+            if (typeof event !== 'undefined') {
             event.preventDefault();
```

Comment:
Very, very minor nitpick - please indent lines 3783 and 3784, now that these are part of an if-block. :-)